documents/PRIVACY.md

```diff
@@ -0,0 +1,19 @@
+
+# Privacy Policy
```

Comment:
Please add the word Placeholder and some mention that the policy will depend greatly on the final client for the solution.